metadata/test/src/main.rs

```diff
@@ -0,0 +1,118 @@
+use solana_client::rpc_client::RpcClient;
+use solana_program::program_pack::Pack;
+use solana_sdk::{
+    pubkey::Pubkey,
+    signature::{read_keypair_file, Signer},
+    system_instruction::create_account,
+    transaction::Transaction,
+};
+use spl_metadata::{
+    instruction::{create_metadata_accounts, init_metadata_accounts, update_metadata_accounts},
+    state::metadata::{Metadata, NAME_LENGTH, SYMBOL_LENGTH},
+    state::PREFIX,
+};
+use spl_token::{instruction::initialize_mint, state::Mint};
+use std::str::FromStr;
+// -------- UPDATE START -------
+
+const KEYPAIR_PATH: &str = "/your/path";
+const METADATA_PROGRAM_PUBKEY_PATH: &str = "/your/path";
+const NEW_MINT_PATH: &str = "/your/path";
+const TOKEN_PROGRAM_PUBKEY: &str = "TokenkegQfeZyiNwAJbNbGKPFXCWuBvf9Ss623VQ5DA";
+const CLUSTER_ADDRESS: &str = "https://api.mainnet-beta.solana.com";
```

Comment:
Can you please employ clap.rs, making the user edit the test code itself to run the test isn't ideal.

Here's something that'll bootstrap this: https://github.com/mvines/solana-cli-template